composer.json

```diff
@@ -6,7 +6,7 @@
     },
     "require": {
         "php": ">=5.5.0",
-        "nette/nette": "2.3.*",
+        "nette/nette": "~2.3.0",
```

Comment:
please use just "~2.3", without .0 (would change semantics and actually allow only 2.3.0-2.3.99999)